build/configs/rtl8730e/hello/defconfig

```diff
@@ -273,27 +280,42 @@ CONFIG_AMEBASMART_FLASH_BASE=0x0
 CONFIG_AMEBASMART_FLASH_CAPACITY=0x800000
 CONFIG_AMEBASMART_FLASH_PAGE_SIZE=256
 CONFIG_AMEBASMART_FLASH_BLOCK_SIZE=4096
-CONFIG_AMEBASMART_SECURE_WORLD_ADDRESS=0x70383020
+#CONFIG_AMEBASMART_SECURE_WORLD_ADDRESS=
 CONFIG_ARCH_BOARD_HAVE_FLASH=y
 
 #
 # Board-Partition Options
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="48,76,4,3260,3260,512,1024,8,"
-CONFIG_FLASH_PART_TYPE="none,none,none,kernel,kernel,smartfs,ftl,bootparam,"
-CONFIG_FLASH_PART_NAME="bl1,reserved,system_data,kernel,kernel,userfs,reserved,bootparam,"
+#CONFIG_FLASH_PART_SIZE="48,76,4,3260,3260,512,1024,8,"
+#CONFIG_FLASH_PART_TYPE="none,none,none,kernel,kernel,smartfs,ftl,bootparam,"
+#CONFIG_FLASH_PART_NAME="bl1,reserved,system_data,kernel,kernel,userfs,reserved,bootparam,"
+CONFIG_FLASH_PART_SIZE="128,1792,6264,8,"
+CONFIG_FLASH_PART_TYPE="none,kernel,none,bootparam,"
+CONFIG_FLASH_PART_NAME="bl1,kernel,reserved,bootparam,"
```

Comment:
1. Could you let me know why you need to modify like this?
2. Where is the secure storage partition?
3. Let's keep `userfs, reserved, bootparam,` at the end with size `512, 1024, 8,`.

Reply:
Hi @sunghan-chang, this is the temporary flash layout design. 
We haven't decided on the final flash layout.
We will take note of your point 3 for the final flash layout.

Thank You.